docs/source/integrations/ios-fairplay-sdk.md

```diff
@@ -1,138 +1,140 @@
- 
+
 The current example initialises an AVAssetDownloadURLSession with a URLSessionConfiguration upon
 loading the ViewController:
- 
+
 backgroundConfiguration = URLSessionConfiguration.background(withIdentifier: "AssetDownloadConfigurationIdentifier")
-        	assetUrlSession = AVAssetDownloadURLSession(configuration: backgroundConfiguration!,
+assetUrlSession = AVAssetDownloadURLSession(configuration: backgroundConfiguration!,
 assetDownloadDelegate: self, delegateQueue: .main)
- 
+
 The asset configuration is then assigned upon use of the download or
 play buttons. To play an asset we create the assetDownloadTask and then assign the associated urlAsset to a player:
- 
+
 let asset = AVURLAsset(url: assetURL!)
 self.drm = vudrmFairPlay(asset: asset, assetName: assetName, token: token)
-               assetDownloadTask = assetUrlSession.makeAssetDownloadTask(asset: asset, assetTitle: assetName,
+assetDownloadTask = assetUrlSession.makeAssetDownloadTask(asset: asset, assetTitle: assetName,
 assetArtworkData: nil, options: nil)!
 assetDownloadTask.taskDescription = assetName
- 
+
 let playerItem = AVPlayerItem(asset: assetDownloadTask.urlAsset)
-       	let player = AVPlayer(playerItem: playerItem)
-       	let playerViewController = AVPlayerViewController()
-       	playerViewController.player = player
-       	present(playerViewController, animated: true) {
+let player = AVPlayer(playerItem: playerItem)
+let playerViewController = AVPlayerViewController()
+playerViewController.player = player
+present(playerViewController, animated: true) {
 player.play()
 }
 And to download, instead of assigning the asset to a player, call:
- 
+
 assetDownloadTask.resume().
- 
+
 To cancel a download, call:
- 
+
 assetDownloadTask.cancel().
- 
- 
+
 Legacy methods:
- 
+
 These are the originally provided methods which are for online playback use only, and are provided for ease of upgrade for legacy users. These methods now also require an asset ID in the initialiser to be compatible with the new framework.
- 
-Legacy method 1: Pass in a reference to your AVPlayer instance, the token to be used, and the asset ID when retrieving a license. 
- 
+
+Legacy method 1: Pass in a reference to your AVPlayer instance, the token to be used, and the asset ID when retrieving a license.
+
 let player = AVPlayer(url: assetURL!)
 self.drm = vudrmFairPlay(player: player, token: token, assetName: assetName)
-            
-Legacy method 2: Pass in a reference to your stream URL and the token to be used when retrieving a license. An AVAsset will be created by VUDRMFairPlay and this can be used to create your player. 
- 
+  
+Legacy method 2: Pass in a reference to your stream URL and the token to be used when retrieving a license. An AVAsset will be created by VUDRMFairPlay and this can be used to create your player.
+
```

Comment:
Surround code snippet in triple back-ticks

Reply:
any changes to this document will have to be done by Adam